backends/graphics/openglsdl/openglsdl-graphics.cpp

```diff
@@ -362,8 +385,8 @@ void OpenGLSdlGraphicsManager::notifyResize(const int width, const int height) {
 	if (ConfMan.getBool("force_resize", Common::ConfigManager::kApplicationDomain)) {
 		createOrUpdateWindow(currentWidth, currentHeight, 0);
 	}
-
-	handleResize(currentWidth, currentHeight);
+	if (!_lockedScreen)
```

Comment:
This doesn't look good.
Even if the window is not resizable, it could be resized by the window manager.